ansible/roles_ocp_workloads/ocp4_workload_external_odf/tasks/workload.yml

```diff
@@ -1,16 +1,12 @@
+# vim: ft=yaml
 ---
 - name: Install Operator and configure external Ceph
-  environment:
-    KUBECONFIG: /home/{{ ansible_user }}/{{ cluster_name }}/auth/kubeconfig
   block:
     - name: Set up ODF operator
       kubernetes.core.k8s:
         state: present
         definition: "{{ lookup('template',  item ) | from_yaml }}"
-      loop:
-        - odf-ns.yaml
-        - odf-og.yaml
-        - odf-sub.yaml.j2
+      loop: [odf-ns.yaml, odf-og.yaml, odf-sub.yaml.j2]
```

Comment:
Are these files or templates? If they're files I usually split the task into a lookup files and a lookup templates task.